ocaml/xapi/message_forwarding.ml

```diff
@@ -3110,6 +3110,11 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
       let local_fn = Local.PIF.set_property ~self ~name ~value in
       do_op_on ~local_fn ~__context ~host (fun session_id rpc -> Client.PIF.set_property rpc session_id self name value)
 
+    let set_disallow_unplug ~__context ~self ~value =
+      let pif_uuid = Db.PIF.get_uuid ~__context ~self in
```

Comment:
I think it's best to stick the the convention here and only print the reference, this is what we seem to do everywhere else in message forwarding - for me, the inconsistency would be more confusing when looking at the logs, because I expect message-forwarding to log refs not UUIDs.